packages/@aws-cdk/aws-stepfunctions-tasks/test/codebuild/private/utils.test.ts

```diff
@@ -0,0 +1,40 @@
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Stack, Aws } from '@aws-cdk/core';
+import { generatePolicyResource, generateSinglePolicyResource } from '../../../lib/codebuild/private/utils';
+
+let stack: Stack;
+let buildIds: string[];
+let buildId: string;
+
+beforeEach(() => {
+  //GIVEN
+  stack = new Stack();
+  buildIds = ['arn:aws:codebuild:us-east-1:123456789012:build/myProject1:buildId',
+    'arn:aws:codebuild:us-east-1:123456789012:build/myProject2:buildId'];
+  buildId = 'arn:aws:codebuild:us-east-1:123456789012:build/myProject:buildId';
+});
+
+test('Generate policy resource for list of build Ids, expect list of project ARNs', () => {
+  let out = generatePolicyResource(stack, buildIds, 'codebuild', 'project');
```

Comment:
these can be `const` as it doesn't change in value during execution. (applies to file)